docs/azure_arc_jumpstart/azure_arc_k8s/alibaba/alibaba_terraform/_index.md

```diff
@@ -0,0 +1,201 @@
+
+  ```shell
+  az extension add --name connectedk8s
+  az extension add --name k8s-configuration
+  ```
+
+  > **Note: If you already used this guide before and/or have the extensions installed, use the bellow commands:**
+
+  ```shell
+  az extension update --name connectedk8s
+  az extension update --name k8s-configuration
+  ```
+
+### Create a new Alibaba Cloud access key
+
+* [Sign up to or have an Alibaba Cloud account](https://www.alibabacloud.com/) - domestic or international
+
+  > **Note: The screenshots and sample outputs shown here are based on a domestic account.**
+
+* Browse to <https://console.aliyun.com/> and login with your Alibaba Cloud account. Once logged in, create a new access key from your profile. After creating it, be sure to copy down the access and secret key.
+
+  ![Alibaba Cloud new access key](./01.png)
+
+  ![Alibaba Cloud new access key](./02.png)
+
+  ![Alibaba Cloud new access key](./03.png)
+
+* Navigate to product *Container Service - Kubernetes*, activate this service and if demanded **Confirm Cloud Resource Access Authorization**
+
+  ![Cloud Resource Access Authorization](./04.png)
+
+## Deployment
+
+The only thing you need to do before executing the Terraform plan is to export the environment variables which will be used by the plan. This is based on the Azure service principal you've just created and your subscription.  
+
+* Export the environment variables needed for the Terraform plan.
+
+  ```shell
+  export ALICLOUD_ACCESS_KEY="<Your Alibaba Cloud access key>"
+  export ALICLOUD_SECRET_KEY="<Your Alibaba Cloud secret key>"
+  export ALICLOUD_REGION="<Alibaba Cloud region>"
+  export ALICLOUD_PROFILE="<Aliyun CLI profile>"
+  export AZURE_SUBSCRIPTION_ID="<Your Azure subscription id>"
+  export AZURE_LOCATION="<Azure location>"
+  export AZURE_RESOURCE_GROUP="<Azure resource group>"
+  export AZURE_CLUSTER_NAME="<Azure Arc cluster name>"
+  ```
+
+  For example:
+
+  ```shell
+  export ALICLOUD_ACCESS_KEY="aBcDeFgHiJkLmNo"
+  export ALICLOUD_SECRET_KEY="aBcDeFgHiJkLmNoPqRtUvWxYz"
+  export ALICLOUD_REGION="cn-shanghai"
+  export ALICLOUD_PROFILE="akProfile"
+  export AZURE_SUBSCRIPTION_ID="XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+  export AZURE_LOCATION="southeastasia"
+  export AZURE_RESOURCE_GROUP="Azure-Arc-Aliyun-K8S"
+  export AZURE_CLUSTER_NAME="azure-arc-aliyun"
+  ```
+
+  > **Note: If you are running in a PowerShell environment, to set the Terraform environment variables, use the _$env:variable-name=variable-value_ (see example below)**
+
+  ```powershell
+  $env:ALICLOUD_ACCESS_KEY="aBcDeFgHiJkLmNo"
+  ```
+
+* From folder `azure_arc_k8s_jumpstart/alibaba/terraform` run the ```terraform init``` command which will download the required terraform providers.
+
+  ![terraform init output](./05.png)
+
+* Deploy cluster by running the ```terraform apply --auto-approve``` command. Wait for the plan to finish.
+
+  ![terraform deploy output](./06.png)
+  ...
+  ![terraform deploy output](./07.png)
```

Comment:
Need to fix all the deprecation messages in the output

Reply:
@likamrat The deprecation messages are caused by the downstream Terraform module. So I see 2 options here:
1. resolve the module into own code and fix the issue which causes more/not so concise code on our end
2. limit the screen shot to "Apply complete! Resources: 8 added..."

Reply:
@KaiWalter I understand. Then I would leave the screenshot the way it is and will put a note on how this can be safely ignored.